Source/ZoomNet.UnitTests/WebhookParserTests.cs

```diff
@@ -344,6 +415,76 @@ public void MeetingServiceIssue()
 			parsedMeeting.Timezone.ShouldBe("America/Los_Angeles");
 		}
 
+		[Fact]
+		public void RecordingCompleted()
+		{
+			var parsedEvent = (RecordingCompletedEvent)new WebhookParser().ParseEventWebhook(RECORDING_COMPLETED_WEBHOOK);
+
+			parsedEvent.EventType.ShouldBe(EventType.RecordingCompleted);
+			parsedEvent.Timestamp.ShouldBe(1626230691572.FromUnixTime(Internal.UnixTimePrecision.Milliseconds));
+			parsedEvent.DownloadToken.ShouldBe("abJhbGciOiJIUzUxMiJ9.eyJpc3MiOiJodHRwczovL2V2ZW50Lnpvb20udXMiLCJhY2NvdW50SWQiOiJNdDZzdjR1MFRBeVBrd2dzTDJseGlBIiwiYXVkIjoiaHR0cHM6Ly9vYXV0aC56b29tLnVzIiwibWlkIjoieFp3SEc0c3BRU2VuekdZWG16dnpiUT09IiwiZXhwIjoxNjI2MTM5NTA3LCJ1c2VySWQiOiJEWUhyZHBqclMzdWFPZjdkUGtrZzh3In0.a6KetiC6BlkDhf1dP4KBGUE1bb2brMeraoD45yhFx0eSSSTFdkHQnsKmlJQ-hdo9Zy-4vQw3rOxlyoHv583JyZ");
+			parsedEvent.AccountId.ShouldBe("AAAAAABBBB");
+			parsedEvent.Recording.ShouldNotBeNull();
+			parsedEvent.Recording.Id.ShouldBe(1234567890);
+			parsedEvent.Recording.Uuid.ShouldBe("4444AAAiAAAAAiAiAiiAii==");
+			parsedEvent.Recording.HostId.ShouldBe("x1yCzABCDEfg23HiJKl4mN");
+			parsedEvent.Recording.AccountId.ShouldBe("x1yCzABCDEfg23HiJKl4mN");
+			parsedEvent.Recording.Topic.ShouldBe("My Personal Recording");
+			parsedEvent.Recording.StartTime.ShouldBe(new DateTime(2021,7,13,21,44,51, DateTimeKind.Utc));
+			parsedEvent.Recording.Password.ShouldBe("132456");
+			parsedEvent.Recording.Duration.ShouldBe(60);
+			parsedEvent.Recording.ShareUrl.ShouldBe("https://example.com");
+			//parsedEvent.Recording.RecordingType.ShouldBe(RecordingType.Personal); // property not implemented
+			parsedEvent.Recording.TotalSize.ShouldBe(3328371);
+			//parsedEvent.Recording.RecordingCount.ShouldBe(2); // property not implemented
+			parsedEvent.Recording.RecordingFiles.ShouldNotBeNull();
+
+			var audioFile = parsedEvent.Recording.RecordingFiles
+				.FirstOrDefault(f => f.ContentType == RecordingContentType.AudioOnly);
+
+			audioFile.ShouldNotBeNull();
+			audioFile.Id.ShouldBe("ed6c2f27-2ae7-42f4-b3d0-835b493e4fa8");
+			audioFile.MeetingId.ShouldBe("098765ABCD");
+			audioFile.StartTime.ShouldBe(new DateTime(2021, 3, 23, 22, 14, 57, 0, DateTimeKind.Utc)); 
+			audioFile.EndTime.ShouldBe(new DateTime(2021, 3, 23, 23, 15, 41, 0, DateTimeKind.Utc)); // property not implemented
+			audioFile.ContentType.ShouldBe(RecordingContentType.AudioOnly);
+			audioFile.FileType.ShouldBe(RecordingFileType.Audio);
+			audioFile.Size.ShouldBe(246560); 
+			//audioFile.FileExtension.ShouldBe("M4A"); // property not implemented
```

Comment:
Thanks for pointing this out. I'll raise an issue to track the fact that we should add this field

Reply:
Addressing this in the files I am about to push